thirdparty_src_checksums.txt

```diff
@@ -17,6 +17,8 @@ f28359aeba12f30d73d9e4711ef356dc842886968112162bc73002645139c39c  glog-0.4.0.tar
 58a6f4277ca2bc8565222b3bbd58a177609e9c488e8a72649359ba51450db7d8  gmock-1.8.0.tar.gz
 1ee8c8699a0eff6b6a203e59b43330536b22bbcbe6448f54c7091e5efb0763c9  gperftools-2.7.tar.gz
 240deacdd628b6459671b83eb0c4db8e97baadf659f25b92e9a078d536bd513e  gperftools-2.8.tar.gz
+12f07a8ba447f12a3ae15e6e3a6ad74de35163b787c0c7b76288d7395f2f74e0  gperftools-2.8.1.tar.gz
+ea566e528605befb830671e359118c2da718f721c27225cbbc93858c7520fee3  gperftools-2.9.1.tar.gz
```

Comment:
do we need all of these here? or are they noops, just in case we upgrade/downgrade?

Reply:
It is OK to keep them here.